pkgs/development/libraries/libpsl/default.nix

```diff
@@ -41,7 +41,10 @@ stdenv.mkDerivation rec {
     libidn2
     libunistring
     libxslt
-  ] ++ lib.optional (!stdenv.hostPlatform.isStatic) python3;
+  ] ++ lib.optional (
+    !stdenv.hostPlatform.isStatic
+    || (!stdenv.hostPlatform.isDarwin || stdenv.buildPlatform == stdenv.hostPlatform)
```

Comment:
This should be `&&`, right? Otherwise static non‐Darwin starts including Python 3.

It might be nice to set `broken` on Darwin cross Python and base this on `python3.meta.broken` instead, since I think the other conditional here is because static Python also broke recently…